python/py-mpi4py/Portfile

```diff
@@ -5,8 +5,7 @@ PortGroup           python 1.0
 PortGroup           bitbucket 1.0
 PortGroup           mpi 1.0
 
-bitbucket.setup     mpi4py mpi4py 3.0.0
-revision            0
```

Comment:
It is now preferred to leave in the `revision            0` line.

Reply:
OK, I amended the commit.